pkg/util/spanlogger/spanlogger.go

```diff
@@ -34,6 +39,9 @@ func New(ctx context.Context, method string, kvps ...interface{}) (*SpanLogger,
 // retrieved with FromContext or FromContextWithFallback.
 func NewWithLogger(ctx context.Context, l log.Logger, method string, kvps ...interface{}) (*SpanLogger, context.Context) {
 	span, ctx := opentracing.StartSpanFromContext(ctx, method)
+	if ids, _ := tenant.TenantIDs(ctx); ids != nil {
+		span.SetTag(TenantIDTagName, tenant.JoinTenantIDs(ids))
```

Comment:
I think it makes sense to use the slice directly instead of a string value and then call the tag "tenant_ids"

Reply:
make sense . done